.travis.yml

```diff
@@ -17,6 +17,7 @@ jobs:
   - stage: Test e2e on privite network
     script: make test-e2e
   - stage: Test e2e on public network
+    if: type != pull_request
```

Comment:
Another solution would be to use `if: fork == false` (there is such [an attribute](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#Left-hand-side)). PRs from the origin would still run this step as the env variables will be passed.

Reply:
I did try `if: fork == false` and they still ran. That build is at https://travis-ci.org/status-im/status-go/jobs/303220190/config

Reply:
Interesting, it looks like a bug. Let's keep `type != pull_request` then 👍

Reply:
Add a comment justifying this line, please. It's non-obvious.